src/ReflectionEngine.php

```diff
@@ -239,7 +239,7 @@ public static function parseFileNamespace($fileName, $namespaceName)
                 continue;
             }
             $topLevelNodeName = $topLevelNode->name ? $topLevelNode->name->toString() : '';
-            if ($topLevelNodeName === $namespaceName) {
+            if ($topLevelNodeName === $namespaceName || "\\{$topLevelNodeName}" === $namespaceName) {
```

Comment:
Then we need to normalize both `$namespaceName` and `$topLevelNodeName` because both developer can supply `$namespaceName` with leading `\` and in analyzed PHP file namespace can be defined as `namespace \NamespaceExample;`.